LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Michael Markl, Paula Wikidal, Marin Brieger, Roman Buchholz, Elias Keis, Maximilian Ammann
```

Comment:
```suggestion
Copyright (c) 2020 Michael Markl, Paula Wikidal, Marvin Brieger, Roman Buchholz, Elias Keis, Maximilian Ammann
```

Reply:
We maybe could also just do something like the following to be safe for the future:
```suggestion
Copyright (c) 2020 The Digitale Ehrenamtskarte contributors
```